### ISSUE TITLE
[Accessibility]: Burndown Chart doesn't have a label

### ISSUE THREAD

**Issue by BolajiOlajide** [sourcegraph/sourcegraph#37466]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34082

### Problem description

When navigating to the burndown chart section with a screen reader, the chart doesn't contain a label to notify the user that they're now in the chart section.

### Expected behavior

The screen reader announces the burndown chart itself when navigating into it.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
